### DISCUSSION TITLE
Not seeing my own tenant

### DISCUSSION THREAD

**Discussion by cinntech** [KelvinTegelaar/CIPP#697] [Q&A] [answered]
Is this by design? I can see all my client Tenants but I can't manage my own?

**Comment by homotechsual** [comment, marked as answer]
Yes, this is by design. You cannot currently see your own tenant. We're working on ways to make this possible in a way which is secure.

**Reply by karnophage** [reply]
If I'm reading this correctly, we should have this ability now? Any update?

**Reply by hhuibers** [reply]
any update, or when can we expect a update about this?

**Comment by justinswall** [comment]
For anyone running across this old discussion, this feature has since been rolled out:

[https://cipp.app/docs/user/gettingstarted/postinstall/owntenant/](https://cipp.app/docs/user/gettingstarted/postinstall/owntenant/)
